db/migrate/20160914191645_add_status_to_requests.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToRequests < ActiveRecord::Migration
+  def change
+    add_column :requests, :status, :integer
```

Comment:
Adrián, yesterday I discussed this with Matias and he suggested taking a more object-oriented approach like the one I implemented. 

The thing is that, even though we know that the first enum is the one mapped to 0, we don't want to use that constant as a default since that mapping might change in the future and the default on the database should be changed too, and other previously created entries. This way, we abstracted from the database and thus any potential changes won't affect the code.